src/components/Bookmark/NoData.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+const NoData = () => {
+  return (
+    <Container>
+      <Content>즐겨찾기 한 영화가 없습니다.</Content>
+    </Container>
+  );
+};
+
+export default NoData;
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
`styles > mixins.ts`쪽의 flexBox를 공통으로 빼두었는데 사용할 수 있을 것 같아요!
```suggestion
  ${({ theme }) => theme.mixins.flexBox()}
```